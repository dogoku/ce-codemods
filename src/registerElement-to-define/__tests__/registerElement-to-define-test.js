'use strict';

const defineSnapshotTests = require('../../../util/defineSnapshotTests');

describe('registerElement-to-define', function() {
  defineSnapshotTests(__dirname);
});
