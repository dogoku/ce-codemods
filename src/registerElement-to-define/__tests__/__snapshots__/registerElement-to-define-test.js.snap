// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`registerElement-to-define snapshot tests exports-constructor 1`] = `
"customElements.define('my-element', MyElement);
module.exports = MyElement;
module.exports.STA_TIC = MyElementProto.STA_TIC;

customElements.define('my-element2', MyElement2);
module.exports = MyElement2;
module.exports.STA_TIC = MyElement2.prototype.STA_TIC;"
`;

exports[`registerElement-to-define snapshot tests exports-proto 1`] = `
"customElements.define('my-element', MyElement);
module.exports = MyElement;

customElements.define('my-element2', MyElement2);
module.exports = MyElement2.prototype;"
`;

exports[`registerElement-to-define snapshot tests no-export 1`] = `
"customElements.define('my-element', MyElement);

customElements.define('my-element', MyElement);"
`;

exports[`registerElement-to-define snapshot tests no-proto 1`] = `
"customElements.define('my-element', MyElement);
module.exports = MyElement;"
`;
